### PR TITLE
Add security:searchApiKeys

### LINKED ISSUE #25: Implement security:searchApiKey
Implement security:searchApiKey

### PR DESCRIPTION
**Pull request by y-abs**
## What does this PR do ?

This PR adds `security:searchApiKeys`

https://github.com/kuzzleio/sdk-dart/issues/25

### How should this be manually tested?

Snippets

### CODE REVIEW

File: doc/2/controllers/security/search-api-keys/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+code: true
+type: page
+title: searchApiKeys
+description: Searches for an user API keys.
```
Comment (scottinet) [inline doc/2/controllers/security/search-api-keys/index.md]:
```suggestion
description: Searches for a user API keys.
```

File: doc/2/controllers/security/search-api-keys/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+# searchApiKeys
+
+<SinceBadge version="7.1.0" />
+
+<SinceBadge version="Kuzzle 2.1.0" />
+
+Searches for an user API keys.
```
Comment (scottinet) [inline doc/2/controllers/security/search-api-keys/index.md]:
```suggestion
Searches for a user API keys.
```

File: doc/2/controllers/security/search-api-keys/snippets/search-api-keys.test.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+name: security#searchApiKeys
+description: Searches for an user API keys
```
Comment (scottinet) [inline doc/2/controllers/security/search-api-keys/snippets/search-api-keys.test.yml]:
```suggestion
description: Searches for a user API keys
```

File: lib/src/controllers/security.dart [hunk 4]
```diff
@@ -533,6 +535,22 @@ class SecurityController extends KuzzleController {
     return KuzzleUser.fromKuzzleResponse(kuzzle, response);
   }
 
+  /// Searches for an user API keys.
```
Comment (scottinet) [inline lib/src/controllers/security.dart]:
```suggestion
  /// Searches for a user API keys.
```

**Review by scottinet — CHANGES_REQUESTED** [review]
grammar

**Review by scottinet — APPROVED** [review]

File: doc/2/controllers/security/search-api-keys/index.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+code: true
+type: page
+title: searchApiKeys
+description: Searches for a user API keys.
+---
+
+# searchApiKeys
+
+<SinceBadge version="7.1.0" />
+
```
Comment (Aschen) [inline doc/2/controllers/security/search-api-keys/index.md]:
```suggestion
```

**Review by Aschen — APPROVED** [review]

**Pull request merged**